.env

```diff
@@ -0,0 +1,7 @@
+DB_HOST=127.0.0.1
```

Comment:
You have to untrack this file from git using git rm.
it's advisable you don't upload the ".env" file.